### ISSUE TITLE
py2app dmg/dylib problems again

### ISSUE THREAD

**Issue by JonnyJD** [JonnyJD/python-discid#43]
The current Picard build (1.3) doesn't work with discid support when packaged with py2app 0.9.
See:
https://gist.github.com/mayhem/e8c3172feca887cd7c2d
http://chatlogs.musicbrainz.org/musicbrainz-devel/2014/2014-10/2014-10-20.html#T22-24-22-939717

The stacktrace is:

```
File "discid/libdiscid.pyo", line 93, in _open_library
10/21/14 12:23:15.925 AM [0x0-0x645645].org.musicbrainz.picard: File "ctypes/__init__.pyo", line 443, in LoadLibrary
10/21/14 12:23:15.925 AM [0x0-0x645645].org.musicbrainz.picard: File "ctypes/__init__.pyo", line 365, in __init__
10/21/14 12:23:15.925 AM [0x0-0x645645].org.musicbrainz.picard: OSError: dlopen(libdiscid.0.dylib, 6): image not found
```

While the dylib is in `/Applications/MusicBrainz Picard.app/Contents/Frameworks/libdiscid.0.dylib`

**Comment by lalinsky** [comment]
Can you please merge this to master and release a new version? Picard can't be built on OS X without this.

**Comment by JonnyJD** [comment]
I am not sure why py2app still fails although this should have been fixed in py2app.
I still can't test it, as I don't have any machine with OS X.

I merged #42 and released python-discid 1.1.1 though.
